README.md

```diff
@@ -61,6 +61,25 @@ Add the overlay to your home.nix (home-manager) or configuration.nix (nixos):
   ];
 }
 ```
+Due to some nixpkgs breaking changes if you are using NixOS 24.05 use the overlay below
+```nix
+{
+  nixpkgs.config = {
+    packageOverrides = pkgs: let
+      pkgs' = import <nixpkgs-unstable> {
```

Comment:
This requires that you have a `nix-channel` named "nixpkgs-unstable". Can you add a comment to clarify? (The exact name "nixpkgs-unstable" does not matter)

Reply:
https://github.com/nix-community/neovim-nightly-overlay/pull/684/commits/0a5c1c8bcfd5cd05b1fe31253b04ece5c618d05c